pkg/apis/garden/types.go

```diff
@@ -910,10 +910,10 @@ type Kube2IAMRole struct {
 	Policy string
 }
 
-// Backup holds information about the backup interval and maximum.
+// Backup holds information about the backup schedule and maximum.
 type Backup struct {
-	// IntervalInSecond defines the interval in seconds how often a backup is taken from etcd.
-	IntervalInSecond int
+	// Schedule defines the cron schedule according to which a backup is taken from etcd.
+	Schedule string
 	// Maximum indicates how many backups should be kept at maximum.
 	Maximum int
```

Comment:
If they are optional then they should be pointers with `// +optional` tag.

Reply:
The `Backup` struct itself is optional, however, the fields inside are not.